src/main/java/com/streamingplatform/moviestreamingplatform/model/dto/mapper/myMapper.java

```diff
@@ -0,0 +1,105 @@
+package com.streamingplatform.moviestreamingplatform.model.dto.mapper;
+
+import com.streamingplatform.moviestreamingplatform.model.Movie;
+import com.streamingplatform.moviestreamingplatform.model.Role;
+import com.streamingplatform.moviestreamingplatform.model.User;
+import com.streamingplatform.moviestreamingplatform.model.dto.MovieDto;
+import com.streamingplatform.moviestreamingplatform.model.dto.RoleDto;
+import com.streamingplatform.moviestreamingplatform.model.dto.UserDto;
+import com.streamingplatform.moviestreamingplatform.model.dto.UserIdDto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.annotation.processing.Generated;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class myMapper implements ImyMapper {
```

Comment:
@Ruleabogdan please rename this into CustomMapper. (Classes must always start with a capital letter)